crates/edr_eth/src/chain_spec.rs

```diff
@@ -9,7 +9,7 @@ use crate::{
 #[derive(Clone, Copy, Debug, Default, PartialEq, Eq, RlpEncodable)]
 pub struct L1ChainSpec;
 
-impl revm::primitives::ChainSpec for L1ChainSpec {
+impl revm::primitives::EvmWiring for L1ChainSpec {
```

Comment:
It doesn't have to be addresses in this PR, just for the future: should we adapt our type names to reflect the name change in `revm`? In this case it'd be `L1EvMWiring` which sounds really weird, so I don't have a strong opinion on this.

Reply:
I don't really like the naming choice that REVM went with as I find it to be an opaque name (not clear what it means). I think `ChainSpec` better reflects that this is the spec of particular chain, so I was going to leave our names as they are.

Reply:
By default, I'd try to match theirs to simplify things, but I agree that their choice isn't great, so I'd leave as is too 👍